classes/wpml/class-wpml-uabb-infolist.php

```diff
@@ -1,16 +1,43 @@
 <?php
-class WPML_UABB_Infolist extends WPML_Beaver_Builder_Module_With_Items {
+/**
+ *  UABB Info List file for WMPL
+ *
+ *  @package UABB Info List WPML Compatibility
+ */
 
+/**
+ * Here WPML_UABB_Infolist extends WPML_Beaver_Builder_Module_With_Items
+ *
+ * @class WPML_UABB_Infolist
+ */
+class WPML_UABB_Infolist extends WPML_Beaver_Builder_Module_With_Items {
+	/**
+	 * Function that renders Info List values
+	 *
+	 * @since 1.6.7
+	 * @param object $settings an object to get values of Info List.
+	 */
 	public function &get_items( $settings ) {
 		return $settings->add_list_item;
 	}
 
+	/**
+	 * Function that renders Info List's fields value
+	 *
+	 * @since 1.6.7
```

Comment:
version number is incorrect in the entire file

Reply:
Added all file correct version number